rust/services/call/host/src/host/error.rs

```diff
@@ -21,42 +19,36 @@ pub enum Error {
 
 #[derive(Error, Debug)]
 pub enum BuilderError {
-    #[error("Multi provider error: {0}")]
+    #[error("Multi provider: {0}")]
```

Comment:
I like this style of skipping redundant `error` in the message. We should probably globally agree on some guidelines on how to concoct errors so that we stay consistent.